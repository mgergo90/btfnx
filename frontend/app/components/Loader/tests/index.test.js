import React from 'react';
import { render } from 'enzyme';

import Loader from '../index';

describe('<Loader />', () => {
  it('Expect to have unit tests specified', () => {
    expect(render(<Loader />)).toMatchSnapshot();
  });
});
