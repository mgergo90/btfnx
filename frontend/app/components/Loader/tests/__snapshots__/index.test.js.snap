// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Expect to have unit tests specified 1`] = `
<div
  class="Loader-root-1"
>
  <div
    class="MuiCircularProgress-root-3 MuiCircularProgress-colorPrimary-6 MuiCircularProgress-indeterminate-5"
    role="progressbar"
    style="width:40px;height:40px"
  >
    <svg
      class="MuiCircularProgress-svg-8"
      viewBox="22 22 44 44"
    >
      <circle
        class="MuiCircularProgress-circle-9 MuiCircularProgress-circleIndeterminate-11"
        cx="44"
        cy="44"
        fill="none"
        r="20.2"
        stroke-width="3.6"
      />
    </svg>
  </div>
</div>
`;
